python/tvm/tir/buffer.py

```diff
@@ -247,7 +247,10 @@ def decl_buffer(
         shape_dtype = shape[0].dtype if hasattr(shape[0], "dtype") else "int32"
         elem_offset = Var("%s_elem_offset" % name, shape_dtype)
     if data is None:
-        data = Var(name, PointerType(PrimType(dtype)), span)
+        # Bool is represented as uint1 in the IR, but stored as uint8
```

Comment:
is this supposed to be "stored as int8"?

Reply:
good catch